core/SConscript.firmware

```diff
@@ -42,12 +42,16 @@ if TREZOR_MODEL in ('1', 'R'):
     FONT_BOLD='Font_PixelOperator_Bold_8'
     FONT_MONO='Font_PixelOperatorMono_Regular_8'
     FONT_BIG='Font_Unifont_Regular_16'
+    FONT_NORMAL_UPPER='Font_PixelOperator_Regular_upper_8'
+    FONT_BOLD_UPPER='Font_PixelOperator_Bold_upper_8'
```

Comment:
With both normal and upper fonts included in this model the glyph data is duplicated, and I don't think the compiler can handle that for us. Both are ~8KiB which I'm not sure we can afford.

Reply:
See comment above.